src/lazy/binary/raw/v1_1/immutable_buffer.rs

```diff
@@ -396,12 +397,11 @@ impl<'a> ImmutableBuffer<'a> {
 
         use OpcodeType::*;
         let result = match opcode.opcode_type {
-            EExpressionWithAddress => {
-                ParseValueExprResult::EExp(self.read_eexp_with_address_in_opcode(opcode))
-            }
-            EExpressionAddressFollows => todo!("eexp address follows"),
-            EExpressionWithLengthPrefix => {
-                ParseValueExprResult::EExp(self.read_eexp_with_length_prefix(opcode))
+            EExpressionWith4BitAddress
+            | EExpressionWith12BitAddress
+            | EExpressionWith20BitAddress
+            | EExpressionWithLengthPrefix => {
+                ParseValueExprResult::EExp(self.read_e_expression(opcode))
```

Comment:
🗺️ I've been considering adding a less granular `OpcodeKind` or similar alongside `OpcodeType` to capture high-level groups like `EExpression` or `Value` for places like this where we don't actually care about the encoding just yet.